src/sentry/api/endpoints/organization_details.py

```diff
@@ -326,6 +327,23 @@ def validate_projectRateLimit(self, value):
             )
         return value
 
+    def validate_aggregatedDataConsent(self, value):
+        request = self.context["request"]
+        if not request.access.has_scope("org:billing"):
+            raise serializers.ValidationError("You do not have permission to change this option")
+        return value
+
+    def validate_genAIConsent(self, value):
+        request = self.context["request"]
+        if not request.access.has_scope("org:billing"):
+            raise serializers.ValidationError("You do not have permission to change this option")
+
+        if get_local_region().name != "us":
```

Comment:
Beyond scope of this PR, but we probably should make region IDs into some kind of StrEnum or something like that, so it would use `RegionEnum.US` instead of `"us"`.